frontend/sportsmatch-app/src/styles/MatchInProgress.css

```diff
@@ -0,0 +1,48 @@
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@700&display=swap');
+
+.host {
+  margin-top: 4em;
+}
+
+.host .row {
+  margin: 0.75em 0;
```

Comment:
If I am not mistaken, MatchInProgress and upcoming are almost the same. I would use just one CSS to avoid duplicated code